src/server.rs

```diff
@@ -268,13 +271,14 @@ impl HttpServer {
     ///
     /// # Errors
     /// Returns an `IOError` when binding or `epoll::create` fails.
-    pub fn new<P: AsRef<Path>>(path_to_socket: P) -> Result<Self> {
+    pub fn new<P: AsRef<Path>>(path_to_socket: P, maybe_max_len: Option<usize>) -> Result<Self> {
```

Comment:
Needs to be more explicit as this is the server constructor function. Something like `request_payload_size_limit` or `request_payload_max_size`. I think it's ok to make the name longer to be more explicit here.